browser/src/shared.scss

```
@@ -25,6 +24,11 @@ $body-color-dark: #f2f4f8;
 
 .command-palette-button {
     align-self: center;
+
+    > span.command-list-popover-button {
```

This looks like an overly specific selector.

```suggestion
    > .command-list-popover-button {
```

Does it have to be nested to be unique enough?